### PR TITLE
UPDATED LOOPBACK SERVICES

### PR DESCRIPTION
**Pull request by ranjit-redekar**

### CODE REVIEW

File: server/server/datasources.json [hunk 1]
```diff
@@ -2,5 +2,25 @@
   "db": {
     "name": "db",
     "connector": "memory"
+  },
+  "UniteMySQL": {
```
Comment (coolbung) [inline server/server/datasources.json]: Is this the right approach - to commit this information into the repo ? It should be set in environment vars ?

File: server/server/model-config.json [hunk 2]
```diff
@@ -14,33 +14,49 @@
     ]
   },
   "User": {
-    "dataSource": "db"
+    "dataSource": "uniteMongoDB"
```
Comment (coolbung) [inline server/server/model-config.json]: Is it necessary to define the datasource for each model config ? Changing it in the future will be hard with this approach.

Comment (ranjit-redekar) [reply server/server/model-config.json]: When we create loopback model then it auto-generate this. If we did some changes and try to create some other model using loopback CLI then it overrides the file.

File: server/server/datasources.json (hunk 1 above)
Comment (ranjit-redekar) [reply server/server/datasources.json]: changes were done please verify it

**Pull request merged**